src/rt/trace.d

```diff
@@ -163,18 +163,23 @@ private int sympair_cmp(in void* e1, in void* e2)
     auto psp1 = cast(SymPair**)e1;
     auto psp2 = cast(SymPair**)e2;
 
-    return (*psp2).count - (*psp1).count;
+    long cmp = (*psp2).count - (*psp1).count;
```

Comment:
Doesn't this pattern produce the wrong results in some cases?  Isn't it better to just use `if (a < b) ...` and let the compiler optimize it?